.travis.yml

```diff
@@ -5,9 +5,6 @@ jdk:
 - openjdk11
 env:
   global:
-  # Following encrypted environment variables must be present for successful deployment
-  # DEV_ACCOUNT_ID,DEV_ACCESS_KEY_ID,DEV_SECRET_ACCESS_KEY
-  # PROD_ACCOUNT_ID,PROD_ACCESS_KEY_ID,PROD_SECRET_ACCESS_KEY
```

Comment:
Maybe we should keep this comment? Or document it somewhere.

Reply:
Reverted this change